README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
 * Maeve Murphy
 * Reagan Smith
 
-## MVP objective
+## Architecture
+
+All data processing is completed using Python 3. Ruby will use the Python scripts to display the resulting graphs on web pages, titled "Overall Distribution" and "Individual Statistics."
```

Comment:
This seems a bit vague/brief. Should we expand a bit more in terms of what Python 3 libraries we're using and how they're helpful?